### PR TITLE
[ADD] Point of Sale: New doc about pricelists

### PR DESCRIPTION
**Pull request by Lau-Odoo**

### CODE REVIEW

File: point_of_sale/advanced_pricing_features/pricelists.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+You probably know the concept of happy hour. During a certain period of
+time, the barman gives a discount on some drinks (usually 50% off or a
+buy one get one free). When the period is over, everything goes back to
+normal. But what is the link with Odoo?
+
+In Odoo, you can set up happy hours. It’s one of the many possible uses
+of *Pricelists*. Those *Pricelists* will allow the creation of
+multiple prices for the same product: a regular one and a special one
+for happy hours. Available in the *PoS* app, those are really
+convenient.
```
Comment (kbr-odoo) [inline point_of_sale/advanced_pricing_features/pricelists.rst]:
```suggestion
You probably know the concept of happy hour: during a certain period of
time, the bar gives discounts on drinks (usually 50% off or a
buy one get one free). When the time is over, prices go back to
normal. But how does that relate with Odoo?
Odoo allows the creation of pricelists, which could be used to set up happy hours times. 
*Pricelists* allow the creation of multiple prices for the same product: a regular one and a special one
for happy hours, for example, and they can be used in the *PoS* app.
```

File: point_of_sale/advanced_pricing_features/pricelists.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+To set up a *Pricelist*, go to :menuselection:`Point of Sale --> Configuration --> Configuration` 
+and enable the *Pricelist* feature. Then, go to :menuselection:`Point of Sale --> Configuration 
+--> Point of Sale` and enable the *Pricelist* for the *PoS*.
+
+.. image:: media/pricelists_01.png
+   :align: center
+
+Now, you can create *Pricelists* by clicking on the *Pricelists* link. 
+Then, set it up by choosing the product category you want to include in your happy hour 
+and the discount.
```
Comment (kbr-odoo) [inline point_of_sale/advanced_pricing_features/pricelists.rst]:
```suggestion
Now, click on *Pricelists* and choose the product category you want to include in your happy hour
and the discount.
```

File: point_of_sale/advanced_pricing_features/pricelists.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+   :align: center
+
+Go back to your *PoS* settings and add the Happy Hour pricelist to the
+list. You can even choose a default pricelist if needed.
+
+.. image:: media/pricelists_03.png
+   :align: center
+
+In the *PoS* interface, a new button appears, allowing you to choose
+between the different *pricelists* you added before.
```
Comment (kbr-odoo) [inline point_of_sale/advanced_pricing_features/pricelists.rst]:
```suggestion
From now on, in the *PoS* interface, a new button is available allowing you to choose
among the different *pricelists* options you added before.
```

File: point_of_sale/advanced_pricing_features/pricelists.rst (hunk 2 above)
Comment (Lau-Odoo) [reply point_of_sale/advanced_pricing_features/pricelists.rst]: I do prefer keeping it the way I wrote it :)

File: point_of_sale/advanced_pricing_features/pricelists.rst (hunk 1 above)
Comment (Lau-Odoo) [reply point_of_sale/advanced_pricing_features/pricelists.rst]: I changed the first part based on your suggestion. I made only some changes in the second part, not everything ;)

**Comment by kbr-odoo** [comment]
@Lau-Odoo something wrong here ^^

**Comment by Lau-Odoo** [comment]
I know @kbr-odoo, I'm trying to fix it right now ^^

File: point_of_sale/advanced_pricing_features/pricelists.rst [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+multiple prices for the same product: a regular one and a special one
+for happy hours. Available in the *PoS* app, those are really
+convenient.
+
+Set up Pricelists
+=================
+
+To set up a *Pricelist*, go to :menuselection:`Point of Sale --> Configuration --> Configuration` 
+and enable the *Pricelist* feature. Then, go to :menuselection:`Point of Sale --> Configuration 
+--> Point of Sale` and enable the *Pricelist* for the *PoS*.
```
Comment (kbr-odoo) [inline point_of_sale/advanced_pricing_features/pricelists.rst]:
```suggestion
To set up a *Pricelist*, go to :menuselection:`Point of Sale --> Configuration --> Configuration` 
and enable *Pricelist* . Then, go to :menuselection:`Point of Sale --> Configuration 
--> Point of Sale` and enable *Pricelist* for the *PoS*.
```

File: point_of_sale/advanced_pricing_features/pricelists.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+.. image:: media/pricelists_02.png
+   :align: center
+
+Go back to your *PoS* settings and add the Happy Hour pricelist to the
+list. You can even choose a default pricelist if needed.
+
+.. image:: media/pricelists_03.png
+   :align: center
+
+From now on, in *PoS* interface, a new button is available, allowing you to choose
```
Comment (kbr-odoo) [inline point_of_sale/advanced_pricing_features/pricelists.rst]:
```suggestion
From now on, in the *PoS* interface, a new button is available, allowing you to choose
```

**Pull request merged**